src/CMakeLists.txt

```diff
@@ -75,6 +75,24 @@ set_target_properties(
 	SOVERSION "1.2"
 )
 
+find_program(
+  CLANG_PATH
+  clang-format
+  )
+
+if(CLANG_PATH)
+  message (STATUS "found clang-format: ${CLANG_PATH}")
+  add_custom_target(
+    clangformat
+    COMMAND ${CLANG_PATH}
+    -style=LLVM
```

Comment:
Doesn't this need to be `-style=file` to use the style from the `.clang-format` file?

Reply:
If the parent directories has `.clang-format`, we don't need to speciy it.